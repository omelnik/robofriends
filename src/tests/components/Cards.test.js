import Cards from '../../components/Cards';
import { shallow } from 'enzyme';
import React from 'react';

describe('<Cards />', () => {
  it('expect to render Cards component with mocked robots', () => {
    const mockedRobots = [
      {
        id: 1,
        name: 'Jon Doe',
        username: 'JonJohn',
        email: 'john@gmail.com',
      },
    ];
    expect(shallow(<Cards robots={mockedRobots} />)).toMatchSnapshot();
  });
});
