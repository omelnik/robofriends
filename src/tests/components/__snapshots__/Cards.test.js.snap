// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Cards /> expect to render Cards component with mocked robots 1`] = `
<div>
  <Card
    email="john@gmail.com"
    id={1}
    key="0"
    name="Jon Doe"
  />
</div>
`;
